File: Week 3/Week 3 Template/A/A_reverse.cpp
#include <bits/stdc++.h>
using namespace std;


vector<int> reverse_list(vector<int>& a){
    /*
    * Your code here; you should return a vector that contains the list 'a' in reverse order.
    */
    return {}; // this is just a placeholder
}

int main(){
    int n;
    cin >> n;
    vector<int> a(n);
    for(int i = 0; i < n; i++) cin >> a[i];
    vector<int> ans = reverse_list(a);
    for(int i : ans){
        cout << i << endl;
    }

}

File: Week 3/Week 3 Template/A/A_reverse.py


def reverse(a):
    # Your code to reverse 'a' goes here
    return None # This is just a placeholder; return the reversed list of integers


n = int(input())
a = []
for i in range(n):
    a.append(int(input()))

ans = reverse(a)
for i in ans:
    print(i)
